orbit/pkg/packaging/windows.go

```diff
@@ -197,3 +204,87 @@ func writePowershellInstallerUtilsFile(opt Options, rootPath string) error {
 
 	return nil
 }
+
+// writeManifestXML creates the manifest.xml file used when generating the 'resource.syso' metadata
+// (see writeResourceSyso). Returns the path of the newly created file.
+func writeManifestXML(opt Options, orbitPath string) (string, error) {
+	filePath := filepath.Join(orbitPath, "manifest.xml")
+	var contents bytes.Buffer
+	if err := manifestXMLTemplate.Execute(&contents, opt); err != nil {
+		return "", fmt.Errorf("parsing manifest XML: %w", err)
+	}
+	if err := ioutil.WriteFile(filePath, contents.Bytes(), constant.DefaultFileMode); err != nil {
+		return "", fmt.Errorf("write file: %w", err)
+	}
+	return filePath, nil
+}
+
+// writeVersionInfoJSON creates the versioninfo.json used when generating the 'resource.syso'
+// metadata (see writeResourceSyso). Returns a pointer to a VersionInfo struct with the proper info.
+func writeVersionInfoJSON(opt Options, orbitPath string, manifestPath string) (*goversioninfo.VersionInfo, error) {
+	vParts := strings.Split(opt.Version, ".")
+	if len(vParts) < 3 {
+		return nil, fmt.Errorf("invalid version: %s", opt.Version)
+	}
+
+	// Append a default 'build' number if the version str contains none.
+	if len(vParts) <= 4 {
```

Comment:
if `len(vParts) < 4 { ... }`? `major.minor.patch`

Reply:
Thanks! Addressed in https://github.com/fleetdm/fleet/commit/4c0361bb239432b6b93fa440c884bf1bcd6700f6#diff-95043ac955352f5e6fe86545b3a5cebae5caa643a9e91da888d302e6d1c7fb98R314